common.js

```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
       'all': ['bsp'],
     },
   },
+  'Lotus': {
+    '7 & 7A': {
+      'all': ['cs', 'ssp']
+    },
+    '7 & 7A (948, 997, & 1098 cc)': {
+      'all': ['dp']
+    },
+    'Cortina': {
+      'all': ['csp', 'hs']
+    },
+    'Eclat': {
+      'all': ['cs', 'ssp']
+    },
+    'Elan': {
+      'all': ['cs']
+    },
+    'Elan (Alternate cylinder head: 26RD0703)': {
+      'all': ['dp']
+    },
+    'Elan (RWD)': {
+      'all': ['cs', 'ssp']
+    },
+    'Elan (FWD)': {
+      'all': ['ssp']
+    },
+    'Elise': {
+      'all': ['ssm'],
+      '1996': ['fp'],
+      '1997': ['fp'],
+      '1998': ['fp'],
+      '1999': ['fp'],
+      '2000': ['fp'],
+      '2001': ['fp'],
+      '2002': ['fp'],
+      '2003': ['fp'],
+      '2004': ['fp'],
+      '2005': ['as', 'fp', 'ssp', 'ssr'],
+      '2006': ['as', 'fp', 'ssp', 'ssr'],
+      '2007': ['as', 'fp', 'ssp', 'ssr'],
+      '2008': ['as', 'fp', 'ssp', 'ssr'],
+      '2009': ['as', 'fp', 'ssp', 'ssr'],
+      '2010': ['as', 'fp', 'ssp', 'ssr'],
+      '2011': ['as', 'ssp', 'ssr']
+    },
+    'Elise SC': {
+      '2008': ['ss'],
+      '2009': ['ss'],
+      '2010': ['ss'],
+      '2011': ['ss']
+    },
+    'Elite': {
+      'all': ['cs']
+    },
+    'Elite 2+2': {
+      'all': ['ssp']
+    },
+    'Elite (1216 cc)': {
+      'all': ['csp']
+    },
+    'Esprit': {
+      'all': ['cs', 'ssm']
+    },
+    'Esprit (4-cyl, V8)': {
```

Comment:
Remove this model and add `ssp` to both the `Esprit Turbo` and the `Esprit` please